runner.js

```diff
@@ -12,12 +12,16 @@ const { analyzeDataPoint } = require("./engine/analyzer.js");
 const logger = require("./logger.js");
 
 // validate_positions_db
+const position = {
+  id: process.env.POSITION_ID,
+  chain: process.env.CHAIN_ID,
+};
 
 const init = async () => {
-  saveOrValidateInitPositionInfo(parseInt(process.env.POSITION_ID));
+  await saveOrValidateInitPositionInfo(position);
   logger.info("init", "done");
   await notify(
-    `unihedge Bot is up and running for position ${process.env.POSITION_ID}`,
+    `unihedge Bot is up and running for position ${position.id} on chain ${position.chain}`,
```

Comment:
We'd better write the chain name rather than numerical chain ID. We can leave for another ticket if it's an unrelated hustle.